### PR TITLE
[1LP][RFR] Automate: test_reports_generate_custom_conditional_filter_report

### PR DESCRIPTION
**Pull request by valaparthvi**
## Purpose or Intent
- __Adding tests__ 
    1. Automate: `test_reports_generate_custom_conditional_filter_report` and remove manual test.

### PRT Run
{{ pytest: cfme/tests/intelligence/reports/test_reports.py -k "test_reports_generate_custom_conditional_filter_report" -vvv }}

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit abbd3fb188ab10c5aed23d20b1c18f26574b961c

The *local* fixture **`edit_service_name`** is used in the following files:
- **cfme/tests/intelligence/reports/test_reports.py**

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit d848bf53656a16f13f0e586fddf06f814edd2ea3

The *local* fixture **`edit_service_name`** is used in the following files:
- **cfme/tests/intelligence/reports/test_reports.py**

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 0e02a536a9403a137c2e853e1a10a56415061179

The *local* fixture **`edit_service_name`** is used in the following files:
- **cfme/tests/intelligence/reports/test_reports.py**

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/tests/intelligence/reports/test_reports.py [hunk 1]
```diff
@@ -187,6 +189,15 @@ def setup_vm(configure_fleecing, appliance, provider):
 
     vm.cleanup_on_provider()
 
+
+@pytest.fixture
+def edit_service_name(service_vm):
```
Comment (digitronik) [inline cfme/tests/intelligence/reports/test_reports.py]: optional: I would make this an part of test only

**Pull request merged**